tests/test_generator.py

```diff
@@ -0,0 +1,41 @@
+import os
+import os.path as osp
+from unittest import TestCase
+
+import datumaro.util.image as image_module
+from datumaro.plugins.synthetic_data import ImageGenerator
+from datumaro.util.test_utils import TestDir
+
+from .requirements import Requirements, mark_requirement
+
+
+class ImageGeneratorTest(TestCase):
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
+    def test_save_image_can_create_dir(self):
+        with TestDir() as test_dir:
```

Comment:
The tests are going to re-download models each time?

Reply:
No, if you ran tests from the same dir